apiary_v2.apib

```diff
@@ -1131,7 +1237,32 @@ This endpoint can be used to retrieve a paginated list of the individual items i
     + Body
 
           {
-          }
+            "count": 12,
+            "next": "https://partner.embed.unmade.com/v2/orders/123456/items?page=2",
+            "previous": null,
+            "results": [
+              {
+                "design": "fd792895-1e32-41b1-90af-683f0cf9f753",
+                "quantity": 1,
+                "partner_data": {
+                  "UNMADE_SOURCE_DESIGN_ID": "fd792895-1e32-41b1-90af-683f0cf9f753",
+                  "UNMADE_ORDER_ITEM_DESIGN_ID": "fd792895-1e32-41b1-90af-683f0cf9f753"
+                },
+                "options": {
+                  "size": "L"
+                },
+                "unit_price": 120.00,
+                "currency": "USD",
+                "reference": "D60033C14A",
+                "factory": "Factory A",
+                "arrive_by_date": "2024-08-11",
+                "ex_factory_date": "2024-07-28",
+                "manufacturing_job": "https://factory.unmade.com/v2/orders/123456/jobs/876543/"
```

Comment:
that's going to be a new relationship .. I didn't realise it before

Reply:
The relationship already exists, it's just implicit. I think it's fine to make it a hard link